app/src/main/java/de/westnordost/streetcomplete/quests/power_attachment/AddPowerAttachment.kt

```diff
@@ -0,0 +1,42 @@
+package de.westnordost.streetcomplete.quests.power_attachment
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.BUILDING
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddPowerAttachment : OsmFilterQuestType<PowerAttachment>() {
+
+    override val elementFilter = """
+        nodes with
+          (power = tower or power = pole or power = insulator)
+          and !line_attachment
+    """
+    override val changesetComment = "Specify line_attachment power support"
+    override val wikiLink = "Key:line_attachment"
+    override val icon = R.drawable.ic_quest_power
```

Comment:
This quest should get a own icon.

Reply:
Why?
SC has similars quests with same the icon.
Do you have a better idea?

Reply:
Because all quests have each a unique icon. But right now, I am unable to come up with a good idea for an icon.

Reply:
Perhaps a circle/disk/line at the bottom (and/or top) of the lightning bolt, making contact?